themes/classic/templates/_partials/head.tpl

```diff
@@ -51,8 +51,8 @@
 {/block}
 
 {block name='head_icons'}
-  <link rel="icon" type="image/vnd.microsoft.icon" href="{$urls.img_ps_url}{$shop.favicon}?{$shop.favicon_update_time}">
-  <link rel="shortcut icon" type="image/x-icon" href="{$urls.img_ps_url}{$shop.favicon}?{$shop.favicon_update_time}">
+  <link rel="icon" type="image/vnd.microsoft.icon" href="{$shop.favicon}?{$shop.favicon_update_time}">
```

Comment:
Won't this break themes using `{$urls.img_ps_url}{$shop.favicon}`?

Reply:
This is the original behaviour, it has been introduced in the 1.7.7.x and we decide to revert it.